includes/Modules/Analytics_4.php

```diff
@@ -186,7 +177,6 @@ public function __construct(
 	) {
 		parent::__construct( $context, $options, $user_options, $authentication, $assets );
 		$this->custom_dimensions_data_available = new Custom_Dimensions_Data_Available( $this->transients );
-		$this->audience_settings                = new Audience_Settings( $this->user_options );
```

Comment:
This is an unrelated tidyup, this `Audience_Settings` instance isn't needed any more.